lib/src/lifecycle_module.dart

```diff
@@ -203,8 +203,8 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
   /// Any [tags] or [references] specified will be added to this span.
   @protected
   void specifyFirstUsefulState({
-    Map<String, dynamic> tags = const {},
-    List<Reference> references = const [],
+    Map<String, dynamic>? tags = const {},
+    List<Reference>? references = const [],
```

Comment:
Looks like these `?` can be removed since there is a default value?
```suggestion
    Map<String, dynamic> tags = const {},
    List<Reference> references = const [],
```

Reply:
Removed `?` and updated test code related to the same